### ISSUE TITLE
"Edit with default SGV editor" is broken on macOS

### ISSUE THREAD

**Issue by Dashcom** [Nebukam/mkfont#23]
When attempting to launch a third party program to edit svg files on macOS, the argument containing the path of the file is malformed.

It does not contain a leading "/" and contains quotes, so instead of being "/path.svg" it is ""path.svg"" which leads nowhere.

I did a (very) ugly fix to get it to open affinity on my machine, maybe it can help you pinpoint the problem.
The "SVG editor executables" field points to a shell script containing this :

```
#!/bin/bash
open "/"`echo $1 | sed s/\"//g`
```

I hope that I was clear enough it's been a while since my last bug report 😅 

BTW your soft is absolutely awesome <3

**Comment by Nebukam** [comment]
Hey @Dashcom !
First of all, thank you so much for taking the time to log this issue, I wasn't aware of it (obviously) and never got to do thorough testing on MacOS as I only had access to Apple hardware for a short while -- I'm glad you have a workaround because...

...since I don't own MacOS hardware, I won't be able to fix this anytime soon on my own ^^'

The issue is not with MKFont itself, but with the underlying app framework (which I also happen to maintain so that's good).
Looking at [this file](https://github.com/Nebukam/nkmjs/blob/main/packages/nkmjs-io-electron/index.js) the last line reads:
`spawn(p_editorPath, ['"${p_targetPath}"'], opts);` 
which clearly add literal quotes to the argument.

Additionally, the path is joined using a windows separator on all platforms, instead of using per-platform sep (Electron, the wrapper on which MKFont is built, _does provide it_).
I also simply didn't knew paths were supposed to have a leading `/` on macOS 

Bottom line, I'll need to update the framework first, then make sure it solve the issue the app, and then find a way to compile on macOS so I can push a fixed release -- it's not a big task, but  the logistic is the issue for me in the short term (can't build  electron for macOS on Windows).
